packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -70,6 +71,14 @@ contract DeployOPChainInput is BaseDeployIO {
         }
     }
 
+    function set(bytes4 _sel, bytes memory _value) public {
```

Comment:
Just for my own edification, why was it necessary to remove the struct typing on the argument?